spec/javascripts/health-care/components/AddressState.spec.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import AddressState from
+    '../../../../_health-care/_js/components/personal-information/AddressState';
+
+describe('<AddressState>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <AddressState state={{ state: 'CA' }} onUserInput={(_unused) => {}}/>
```

Comment:
I agree about renaming `onUserInput` to `onValueChange`. But I'm confused about why we would want to change `state` to `value`. `state` is just the name of the prop on `AddressState` that sets the data from `applicationData`, right? And it looks like the other question components have a prop with a name that corresponds to the question (e.g., `ssn` for `SocialSecurityNumber`). `value` has a very specific meaning to me as the value attribute on an input, which is how you're using it on `ErrorableInput`. Do we want to also use that for question components?